### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by GrahamCampbell**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/qoWlZq).

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Here's a replacement PR with the correct config, and sent from the correct user. Fixed that bug now. :)

File: tests/bootstrap.php [hunk 1]
```diff
@@ -18,6 +18,6 @@
 }
 
 /**
-* @var $loader ClassLoader
-*/
+ * @var ClassLoader
```
Comment (GrahamCampbell) [inline tests/bootstrap.php]: Looks like this is being changed because the phpdoc should be all on one line.

Comment (sstok) [inline tests/bootstrap.php]: Ah so thats the reason why this keeps happening :)

Comment (GrahamCampbell) [inline tests/bootstrap.php]: `/** @var $loader ClassLoader */` should work. :)

Comment (sstok) [inline tests/bootstrap.php]: I applied the change, lets hope it works :) I can squash the PR when merging :+1:

**Comment by sstok** [comment]
Thank you @GrahamCampbell

**Pull request closed without merging**